nx-X11/programs/Xserver/hw/nxagent/Events.c

```diff
@@ -2522,7 +2522,7 @@ FIXME: This can be maybe optimized by consuming the
               nxagentSplashCount == 1 && X -> xexpose.count == 0)
       {
         #ifdef DEBUG
-        fprintf(stderr, "nxagentHandleExposeEvent: Clearing root tile window id [%ld].\n",
```

Comment:
This hunk can also stay.

`nxagentWindowPriv(nxagentRootTileWindow)->window` is passed to XClearWindow, which takes a `Window64` parameter (i.e., `unsigned long`) and implicit conversion rules make sure that the smaller value passed here is promoted to `unsigned long`, so no problem here.